### DISCUSSION TITLE
Obtenir la base de données du site

### DISCUSSION THREAD

**Discussion by matteoP02** [incubateur-ademe/nosgestesclimat#1594] [Q&A]
Bonjour, 
Nous sommes des étudiants en école d'ingénieurs, nous sommes en train de réaliser un projet pour réaliser le bilan carbone des étudiants de notre école. Cependant nous aimerions pouvoir comparer leurs résultats à une base de données crédible. Serait-il donc possible d'obtenir l'accès à votre base de données afin de compléter notre bilan carbone. 
En vous souhaitant une bonne journée.

**Comment by laem** [comment]
Bonjour, nous n'avons pas de base de données :) 

CF https://github.com/datagir/nosgestesclimat-site/issues/400 pour la justification.
